packages/inference/src/types.ts

```diff
@@ -24,6 +24,14 @@ export interface Options {
 	 * (Default: "same-origin"). String | Boolean. Credentials to use for the request. If this is a string, it will be passed straight on. If it's a boolean, true will be "include" and false will not send credentials at all.
 	 */
 	includeCredentials?: string | boolean;
+
+	/**
+	 * The billing account to use for the requests.
+	 *
+	 * By default the requests are billed on the user's account.
+	 * Requests can only be billed to an organization the user is a member of, and billing should be enabled for that organization.
```

Comment:
```suggestion
	 * Requests can only be billed to an Enterprise organization the user is a member of.
```

(If I understood correctly)

Reply:
```suggestion
	 * Requests can only be billed to an organization the user is a member of, and which has subscribed to Enterprise Hub.
```

Reply:
ah yes thanks, didn't know it when I opened the PRs :smile: Aligned the phrasing with https://github.com/huggingface/huggingface_hub/pull/2940

Reply:
Yes, that sounds better indeed.